src/api/messages.ts

```diff
@@ -21,3 +22,28 @@ export const postMessage = async (formData: {
     };
   }
 };
+
+export const getMessages = async (isApproved?: boolean | null) => {
+  const data =
+    (await apiGet<Message[]>(
+      `messages/email/received-owner-requests?approvedFilter=${isApproved}`,
+    )) || [];
+
+  return data;
+};
+
+export const getRequests = async () => {
+  const data =
+    (await apiGet<Message[]>('messages/email/sent-user-requests')) || [];
+
+  return data;
+};
+
+export const patchMessageRequest = async (id: string, approval: Approval) => {
```

Comment:
It does not have to be in format `<method><some-part-of-api-path>` it should, as usual, give the best context it can
```suggestion
export const approveMessageRequest = async (id: string, approval: Approval) => {
```